nixos/tests/plausible.nix

```
@@ -0,0 +1,46 @@
+import ./make-test-python.nix ({ pkgs, lib, ... }: {
+  name = "plausible";
+  meta = with pkgs.stdenv.lib.maintainers; {
```

```suggestion
  meta = with lib.maintainers; {
```

If we already pass in lib we can use it to make the variables shorter.